### ISSUE TITLE
Update gen2 ocr example

### ISSUE THREAD

**Issue by VanDavv** [luxonis/depthai-experiments#60]
I tried to migrate [this example](https://github.com/luxonis/depthai-experiments/tree/master/gen2-ocr) to latest DepthAI Gen2 API (commit hash `45765d01ad9c7d8d9ae99ca0f76c5b03b706a1c4`), and got the following error after running against this version

```
Traceback (most recent call last):
  File "/home/vandavv/dev/luxonis/depthai-experiments/gen2-ocr/main.py", line 12, in <module>
    colorCam.setAutoFocusMode(depthai.RawCameraControl.AutoFocusMode.AUTO)
AttributeError: 'depthai.ColorCamera' object has no attribute 'setAutoFocusMode'
```

**Comment by TannerGilbert** [comment]
Fixed by bec6e6b6c5ab9eba9c2d97cd1f46cbba9e4df22b.

**Comment by VanDavv** [comment]
Thanks!
